estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
You are missing an empty line at the bottom of each file. You should configure your IDE such that it is done automatically. This comment applies to all files in your code.